src/phoenix/server/api/types/EmbeddingDimension.py

```diff
@@ -300,6 +307,133 @@ def UMAPPoints(
             ),
         )
 
+    @strawberry.field
+    def hdbscan_clustering(
```

Comment:
I'd actually not nest this under an Embedding dimension at all. Makes it harder to call and isn't really tied to a dimension at all.

Reply:
yea, i had the same thought, but didn't know what would be a better place. will take a second look